content/getstarted/02-item.md

```diff
@@ -13,9 +13,7 @@ weight = 2
 +++
 [KubeVirt](https://kubevirt.io) is a virtual machine management add-on for [Kubernetes](https://kubernetes.io). It allows users to run VMs alongside containers in the their Kubernetes or [OpenShift](https://www.openshift.com) clusters. This document describes a quick way to deploy either Kubernetes or OpenShift, KubeVirt and [Ember-CSI](https://ember-csi.io).
 
-The [Ember-CSI-Kubevirt](https://github.com/embercsi/ember-csi-kubevirt.git) repository provides a wrapper on the normal KubeVirt workflow and it aims to provide a seamless integration with it while taking care of the Ember-CSI deployment.
-
-To use Ember-CSI plugin on KubeVirt, we will utilize the [Ember-CSI-KubeVirt repository](https://github.com/embercsi/ember-csi-kubevirt.git) which can deploy an all-in-one demo deployment. The all-in-one demo deployment comprises of either Kubernetes/OpenShift with KubeVirt, an ephemeral Ceph environment and finally an Ember-CSI deployment configured with the previously deployed ephemeral Ceph plugin.
+The [Ember-CSI-Kubevirt](https://github.com/embercsi/ember-csi-kubevirt.git) repository provides a wrapper on the normal KubeVirt workflow and it aims to provide a seamless integration with it while taking care of the Ember-CSI deployment.i The all-in-one demo deployment created by the Ember-CSI-Kubevirt repository comprises of either Kubernetes or OpenShift with KubeVirt, an ephemeral Ceph cluster and finally an Ember-CSI deployment configured with the previously deployed ephemeral Ceph cluster.
```

Comment:
"deployment.i The" ==> "deployment. The"

Reply:
It would be good to mention that we are using our operator and put a link to it.